ocfweb/docs/docs/services/hpc.md

```diff
@@ -1,65 +1,115 @@
 [[!meta title="High Performance Computing"]]
 
-**NOTE: We are in the process of trialling this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
+**NOTE: We are in the process of trialing this service to users so that we can make the service as accommodating and secure as possible. This means that items concerning the service, including this documentation, are subject to change. We will do our best to keep everyone updated and notified of changes as they come.**
 
 ## Introduction
 
-At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. Our main HPC server is currently named `corruption`, however we also have plans to expand the cluster to maximize the use of the resources at our disposal.
+At the OCF we offer a High Performance Computing (HPC) service for individuals and groups that need to run computationally demanding software. We currently have one main HPC server, which handles all of our HPC jobs. However, have plans to expand the cluster to maximize the use of the resources at our disposal.
 
-### Gaining Access
+## Gaining Access
 
-In order to access the HPC cluster, you must send a proposal to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or group account name and a short, but detailed technical description of the experiments you plan to run on our HPC infrastructure. This would include information about the nature of the software being developed, as well as the amount of computational resources that are expected to be needed.
+In order to access the HPC cluster, please send an access request to [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu). Make sure to include your OCF username or [group account][group] name and a detailed technical description of the projects you plan to run on our HPC infrastructure. This would include information about the nature of the software being run, as well as the amount of computational resources that are expected to be needed.
 
-#### Connecting
+### Connecting
 
-Our Slurm master node is currently `segfault`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
+Our Slurm master node is named `hpcctl`. Once you submit your proposal and are approved access you will be able to connect via ssh by running the following command:
```

Comment:
Well it's not _named_ hpcctl, so that's not technically a correct statement. This paragraph could IMO be better written as

> Once you submit your proposal and are approved access you will be able to connect to our Slurm master node via SSH by running the following command:

We could perhaps even leave out the part about the Slurm master node entirely.